doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3355,6 +3355,128 @@ the conversion in hypotheses :n:`{+ @ident}`.
 
    This is the most general syntax that combines the different variants.
 
+.. tacn:: with_strategy @strategy_level_or_var [ {+ @smart_qualid } ] @ltac_expr3
+   :name: with_strategy
+
+   This tactic allows changing the unfolding behavior of a constant
+   locally within the execution of :n:`@ltac_expr`, analogous to the
+   :cmd:`Strategy` command.  This can be useful for guarding calls to
```

Comment:
How about a more concrete statement along these lines (if correct)?:

```
Executes :token:`ltac_expr3`, applying the alternate unfolding behavior
that :cmd:`Strategy` command controls, but only for :token:`ltac_expr3`.
This can be useful ...
```

Reply:
Sure, thanks, and updated